api/types/plugin.go

```diff
@@ -42,6 +42,10 @@ type PluginConfig struct {
 	// Required: true
 	Description string `json:"Description"`
 
+	// Docker Version used to create the plugin
+	// Required: true
+	DockerVersion string `json:"DockerVersion"`
```

Comment:
Should the new fields be marked `omitempty` so they don't show up as empty values for old plugins?